### PR TITLE
Paper Submission - Srinath - "An Analysis on Genomic Correlation..."

### PR DESCRIPTION
**Pull request by taylorgibson**

### CODE REVIEW

File: papers/2024-morganton-srinath/myst.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      email: srinath24s@ncssm.edu
+  abbreviations:
+    QTL: Quantitative Trait Loci
+    LOD: Logarithm of the Odds
+  funding: 
+    statement: Funding for this program is provided by the North Carolina School of Science and Mathematics, the University of North Carolina General Administration, and the General Assembly for the State of North Carolina.
+  exports:
+  - format: pdf
+    template: arxiv_nips
+    article: article.md
```
Comment (rowanc1) [inline papers/2024-morganton-srinath/myst.yml:32]: Remove this for now.

File: papers/2024-morganton-srinath/myst.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+version: 1
+project:
```
Comment (rowanc1) [inline papers/2024-morganton-srinath/myst.yml:2]: Add DOI ignore statements like this:

https://github.com/scipy-conference/scipy_proceedings/blob/82a60c3fedc6fb3eec6df8c00fe0b7812bc4ae8a/papers/00_myst_template/myst.yml#L36C89-L45C19

This allows the pull-request to go green!

File: papers/2024-morganton-srinath/myst.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    LOD: Logarithm of the Odds
+  funding: 
+    statement: Funding for this program is provided by the North Carolina School of Science and Mathematics, the University of North Carolina General Administration, and the General Assembly for the State of North Carolina.
+  exports:
+  - format: pdf
+    template: arxiv_nips
+    article: article.md
+site:
+  template: article-theme
+  title: An Analysis on Genomic Correlation for Gallstone Susceptibility
```
Comment (rowanc1) [inline papers/2024-morganton-srinath/myst.yml:35]: Not necessary for the title!

**Review by rowanc1 — COMMENTED** [review]
Gave some feedback!

File: papers/2024-morganton-srinath/myst.yml (hunk 1 above)
Comment (taylorgibson) [reply papers/2024-morganton-srinath/myst.yml]: Done!

File: papers/2024-morganton-srinath/myst.yml (hunk 3 above)
Comment (taylorgibson) [reply papers/2024-morganton-srinath/myst.yml]: Done!

File: papers/2024-morganton-srinath/myst.yml (hunk 2 above)
Comment (taylorgibson) [reply papers/2024-morganton-srinath/myst.yml]: Made that change, but it looks like it's still not ignoring the lack of a DOI.

**Comment by taylorgibson** [comment]
@rowanc1 I updated the `.gitignore` file in this latest update to avoid accidentally checking in `.DS_Store` files on my Mac. Unfortunately it looks like since I committed files from outside the project folder it has caused an issue in the automated checks. Before I try to "fix" this on my own, I thought I would ask for advice on best practices to ensure that actually fix this issue moving forward. Thanks!

**Comment by rowanc1** [comment]
If you commit files outside of the paper folder on the main branch that will be best. If you are comfortable with rebasing, that is an easy way to pick up the changes. I think a merge of main into the paper branch should also work (I haven't fully tested that the changes are picked up correctly in that case though).

If you want a video on rebasing, happy to pull one together.

**Comment by taylorgibson** [comment]
@rowanc1 A video on how to do this would be appreciated! I've spent some time over the weekend trying to sort this on my own, but unfortunately outside my area of expertise. Or, if you think just including another commit on this branch that reverts the change I made would solve the issue, that's something I could do.

Thanks!

**Comment by taylorgibson** [comment]
@rowanc1  What's going on with this one? I changed the `myst.yml` file to match the other article, but it's still getting stuck on the DOI requirement? Is that a per article setting, or did I just do something wrong here?

**Comment by rowanc1** [comment]
I have set it up currently to just ignore the DOIs in that list. If you would like to just change that to optional entirely and not worry about DOIs this year we can do that!

This is the check that is failing:

```
    × DOI Exists: Citation does not have DOI: noauthor_gallstones_2021
    × DOI Exists: Citation does not have DOI: myles2008quantitative
    × DOI Exists: Citation does not have DOI: lyons1
    × DOI Exists: Citation does not have DOI: clevelandclinic_gallbladder:_nodate
    × DOI Exists: Citation does not have DOI: broman2003r
    × DOI Exists: Citation does not have DOI: arends2010r
    × DOI Exists: Citation does not have DOI: silver1995mouse
```

So you would add each of these to that list as before:

```
 - noauthor_gallstones_2021
 - etc...
```

**Comment by taylorgibson** [comment]
@rowanc1 let's not enforce DOI's this year, but I'd like to discuss how to do that in the future so we can train up our students to search for materials that have DOI's present when conducting literature review.

**Pull request merged**